paasta_tools/contrib/rightsizer_soaconfigs_update.py

```diff
@@ -157,11 +161,12 @@ class CassandraRightsizerResult(TypedDict):
     suggested_replicas: str
 
 
-class CassandraRecommendation(TypedDict):
+class CassandraRecommendation(TypedDict, total=False):
```

Comment:
just curious: why would some of these keys be missing?

Reply:
The reason keys might be missing is because `keys_to_apply` might only contain a subset of possible keys, e.g. just `["cpus"]`. This happens when the script is executed with `--csv-keys cpus`. This flag makes it possible to update autotuned defaults only for a subset of the specs, rather than tackling everything all the time.